modules/before-updating-ocp.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="before-updating-ocp{context}"]
+= Before updating the {product-title} cluster
+
+Before updating, consider the following:
+
+* You have recently backed up etcd.
+
+* You might experience an interruption during the update process if you run an Operator or you have configured any application with the pod disruption budget. If `minAvailable` is set to 1 in `PodDisruptionBudget`, the nodes are drained to apply pending machine configs which might block the eviction process. If several nodes are rebooted, all the pods might run on only one node, and the `PodDisruptionBudget` field can prevent the node drain.
```

Comment:
We must remove the text `You might experience an interruption during the update process if you run an Operator or you have configured any application with the pod disruption budget` as this is not correct.

Reply:
Updated!